Sources/PayPalDataCollector/MagnesSDKProtocol.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import PPRiskMagnes
+
+protocol MagnesSDKProtocol {
+    func setUpWithParams(_ magnesParams: MagnesSetupParams) throws
+    func collectDeviceData(
+        withPayPalClientMetadataId cmid: String,
+        withAdditionalData additionalData: [String: String]
+    ) throws -> MagnesSDKResult
+}
+
+extension MagnesSDK: MagnesSDKProtocol {
+
+    func setUpWithParams(_ params: MagnesSetupParams) throws {
+        // workaround to avoid max line column width linter error
```

Comment:
I think we can remove this comment since the linter would throw a warning if we tried to single line it in the future